### DISCUSSION TITLE
Heading 4

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#320] [Component suggestions]
<img width="720" alt="visuele weergave van de heading 4 component" src="https://github.com/user-attachments/assets/540d7554-dd2c-43e9-b4a2-db87f147f7cc">

## Naam

Heading 4

## Doel

Toont de titel van een sub-sub sectie en helpt de inhoud van een pagina te structureren.

## Ook bekend als

Heading, Titel

## Gerelateerde componenten

[Heading](https://github.com/orgs/nl-design-system/discussions/220), [Heading 1](https://github.com/orgs/nl-design-system/discussions/317), [Heading 2](https://github.com/orgs/nl-design-system/discussions/318), [Heading 3](https://github.com/orgs/nl-design-system/discussions/319), [Heading 5](https://github.com/orgs/nl-design-system/discussions/321), [Heading 6](https://github.com/orgs/nl-design-system/discussions/322), [Rich Text Content](https://github.com/orgs/nl-design-system/discussions/289)

## Onderzoek

**Comment by Yolijn** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Heading 4 op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=heading+4)
